distribution/lib/Standard/Table/0.0.0-dev/src/Io/File_Format.enso

```diff
@@ -108,6 +124,11 @@ type Delimited
     read file on_problems =
         Delimited_Reader.read_file this file on_problems
 
+    ## Implements the `Table.write` for this `File_Format`.
+    write_table : File -> Table -> Existing_File_Behavior -> Column_Mapping -> Problem_Behavior -> Nothing
+    write_table _ _ _ _ _ =
+        Errors.unimplemented "This feature is not implemented yet."
```

Comment:
If someone throws `unimplemented` then `"This feature is not implemented yet."` doesn't bring much. Maybe 
```suggestion
        Errors.unimplemented "Table.write for Delimited type is not implemented yet."
```
Similarly for other cases.